.github/workflows/build_pr.yml

```diff
@@ -0,0 +1,25 @@
+name: Build for PR
+
+on:
+  pull_request:
+    types: [ opened, synchronize ]
+    branches: [ 'develop', 'ci/upload-builds-to-s3-on-pr', 'feat/run-on-pr' ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions: 
+  checks: write
+
+jobs:
+  build_develop:
+    #if: github.event.pull_request.draft == false
+    uses: ./.github/workflows/_reusable_app_release.yml
+    with:
+      datadog_enabled: false
+      fastlane_action: development_pr
+      is_cloud_build: true
+      pr_s3_path: "PR-${{ github.event.pull_request.number }}"
```

Comment:
Something to check... 

1. PR is opened and a build is uploaded to S3
2. PR is updated and a new build is uploaded to same path

Does this work... E.g. overwriting the previous build with the latest build?

Reply:
It creates a new build with a folder containing the PR number.

Reply:
```
PR
  PR-1793
  - Wire-development-1793_merge-simulator-12038.ipa
  - Wire-development-1793_merge-simulator-12039.ipa
  - Wire-development-1793_merge-simulator-12043.ipa
```

Reply:
we use the same flow as the Android team

Reply:
OK, so the build itself will have a unique name, and therefore we keep each build from that particular PR?

Reply:
@samwyndham  but we talk, we discuss it more during the call

Reply:
I think this is fine. No need to discuss further. I will close my comment.